database/factories/TeamFactory.php

```
@@ -0,0 +1,40 @@
+            'description' => $this->faker->sentence(),
+            'personal_team' => false,
+            'show_boarding' => false,
+        ];
+    }
+
+    /**
+     * Indicate that the team is a personal team.
+     */
+    public function personal(): static
+    {
+        return $this->state(fn (array $attributes) => [
```

PHPMD (2.15.0)

Avoid unused parameters such as '$attributes'. (undefined)

(UnusedFormalParameter)